livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -279,9 +279,12 @@ async def send_task():
                 if isinstance(data, rtc.AudioFrame):
                     # TODO(theomonnom): The remix_and_resample method is low quality
                     # and should be replaced with a continuous resampling
-                    frame = data.remix_and_resample(
-                        self._sample_rate, self._num_channels
-                    )
+                    # frame = data.remix_and_resample(
```

Comment:
This is sort of an anti-pattern but wanted to leave here to check the intention of still reworking this.

Reply:
hey @vanics can we actually keep the remix_and_resample method. If the sample rates match, it's no-op but we don't want it to completely break if users override the default.

Reply:
Actually i'll merge and do a followup. Thanks for making this PR!